technical-vision.md

```diff
@@ -0,0 +1,9 @@
+# OpenSSF Technical Vision
+
+We envision a future where participants in the open source software ecosystem focus on delivering high quality products and services, with security handled proactively, automatically, and as a matter of course.
+- Developers can easily learn secure development practices, and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
+- Developers, auditors and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.
+- Developers and researchers can identify security issues (including unintentional vulnerabilities and malicious software) and have this information swiftly flow backwards through the supply chain to someone who can rapidly address the issue.
+- All stakeholders can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
```

Comment:
Perhaps instead of stakeholders, "The community", some might have no stake, but just want to contibute for reasons for altruism.